auth-service/src/main/kotlin/uk/gov/communities/delta/auth/security/ADLdapLoginService.kt

```diff
@@ -73,6 +75,7 @@ class ADLdapLoginService(
     @Blocking
     private fun ldapBind(userDn: String, password: String): IADLdapLoginService.LdapLoginResult {
         var context: InitialDirContext? = null
+        val span = ldapSpanFactory("AD-ldap-user-bind").startSpan()
         return try {
             context = ldapRepository.bind(userDn, password)
```

Comment:
Does it make sense to do it inside ldapRepository.bind instead? Or are you deliberately not logging every lookup?

Reply:
We'd need another span for `mapUserFromContext` then (my understanding is that's also an LDAP call), and for running the block in `useServiceUserBind`. I think either would be fine but probably not worth changing